### PR TITLE
Adds additional exercises 3 - 5 to cover a range of Java topics

### PR DESCRIPTION
**Pull request by kimdiep**
Adds additional exercises 3 - 5 to cover a range of Java topics

### CODE REVIEW

File: docs/exercise004.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Exercise 004
+
+This exercise gives you the opportunity to explore classes, Date/Time and unit tests in Java.
+
+### Instructions
+Given a moment, your job is to write a program to find the moment that would be after a gigasecond has passed.
```
Comment (eggsy84) [inline docs/exercise004.md]: Hey @kimdiep 
I feel like this one could be explained a little better. Maybe a subtle change to

"Given a moment in time (represented as a date/time value), your job is to....."

**Comment by eggsy84** [comment]
@kimdiep All looks good to me - I like the increasing complexity. Just the one comment on exercise 004

File: docs/exercise004.md (hunk 1 above)
Comment (kimdiep) [reply docs/exercise004.md]: Thanks @eggsy84 - agreed :) Will make the change now :)

Comment (kimdiep) [reply docs/exercise004.md]: Hi @eggsy84 - that change has been implemented now :) If you could re-review, that would be super duper :)

Comment (eggsy84) [reply docs/exercise004.md]: Hey @kimdiep 
Yep that looks good to me. More descriptive on the moment. 👍

**Comment by eggsy84** [comment]
All looks good to me now @kimdiep

**Pull request merged**